apple/HevSocks5Tunnel.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,2393 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 56;
+	objects = {
+
+/* Begin PBXBuildFile section */
+		B603F4B52BE08D6800C025B4 /* hev-tunnel-macos.c in Sources */ = {isa = PBXBuildFile; fileRef = B603F4642BE08D6700C025B4 /* hev-tunnel-macos.c */; };
```

Comment:
I don't think it's necessary to list all the source code files, in my case `build.sh` works just fine.